### ISSUE TITLE
Flaky test io.prestosql.plugin.kudu.TestKuduIntegration*

### ISSUE THREAD

**Issue by ebyhr** [prestosql/presto#3314]
https://github.com/prestosql/presto/runs/554953806
```
[ERROR]   TestKuduIntegrationDecimalColumns>AbstractTestQueryFramework.init:76->createQueryRunner:48 » Runtime
[ERROR]   TestKuduIntegrationHashPartitioning.destroy:42 NullPointer
[ERROR]   TestKuduIntegrationIntegerColumns.destroy:50 NullPointer
[ERROR]   TestKuduIntegrationRangePartitioning.destroy:98 NullPointer
[ERROR]   TestKuduIntegrationSchemaNotExisting.destroy:49 NullPointer
```

**Comment by findepi** [comment]
this failed on me too today

cc @Praveen2112

**Comment by Praveen2112** [comment]
Most of the issues we see isn't related to diskspace ? Like maven check or product tests or kudu ?

**Comment by Praveen2112** [comment]
Fixed by #3315
